src/main.rs

```diff
@@ -130,6 +131,23 @@ fn main() -> anyhow::Result<()> {
         return Ok(());
     }
 
+    let on_exit = || match crossterm::terminal::disable_raw_mode() {
+        Ok(_) => std::process::exit(0),
```

Comment:
according to https://shapeshed.com/unix-exit-codes/ we should use `130` when terminated by <kbd>CTRL</kbd><kbd>C</kbd>